angular/projects/spark-angular/src/lib/components/sprk-tabbed-navigation/sprk-tabbed-navigation.component.ts

```diff
@@ -5,15 +5,12 @@ import {
   ElementRef,
   HostListener,
   Input,
-  QueryList
+  QueryList,
 } from '@angular/core';
+import { Observable } from 'rxjs';
```

Comment:
we dont need to import Observable here, since QueryList has .changes

Reply:
done!